tests/unit_tests/test_wave_06.py

```diff
@@ -109,8 +109,14 @@ def test_swap_best_by_category():
         my_priority="Clothing",
         their_priority="Decor"
     )
+    #assert result == True
+
+    assert len(tai.inventory) == 3
+    assert len(jesse.inventory) == 3
 
-    raise Exception("Complete this test according to comments below.")
+    assert item_f in tai.inventory
```

Comment:
Good job! This checks specifically for the swapped items, but it might also be a good idea to check that every item is where it's supposed to be.